components/popover/popover.vue

```diff
@@ -451,6 +451,19 @@ export default {
       type: Boolean,
       default: false,
     },
+
+    /**
+     * Sets the element to which the popover is going to append to.
+     * @values 'parent', document.body
+     */
+    appendTo: {
+      type: [HTMLElement, String],
+      default: () => document.body,
+      validator: v => {
+        if (v instanceof HTMLElement) return true;
+        return !(typeof v === 'string' && v !== 'parent');
```

Comment:
The same conditional but avoiding negation is simpler

```suggestion
        return typeof v === 'string' && v === 'parent';
```